app/controllers/home_controller.rb

```diff
@@ -1,6 +1,8 @@
 # Controller to handle root HTTP
 # connections and display the home page
 class HomeController < ApplicationController
+  layout 'sul-requests'
```

Comment:
Should we add this to the `ApplicationController` instead so it applies globally to all of our controllers?